test/bd-chains-and-dma-tasks/materialize-bd-chains/good-1.mlir

```diff
@@ -28,7 +28,7 @@ module {
     aiex.runtime_sequence(%arg0: memref<8xi16>, %arg1: memref<12xi16>, %arg2: memref<8xi16>) {
       %t1 = aiex.dma_start_bd_chain @simple_chain(%arg0, %arg1, %arg2) : (memref<8xi16>, memref<12xi16>, memref<8xi16>)  
                                     on (%tile_0_0, MM2S, 0) 
-      // CHECK: %[[task1:[0-9]+]] = aiex.dma_configure_task(%tile_0_0, MM2S, 0) {
+      // CHECK: %[[task1:.+]] = aiex.dma_configure_task(%tile_0_0, MM2S, 0) {
```

Comment:
These changes unrelated to the functionality of this PR make it harder to review

Reply:
Sorry about that, factored out into PR #1698 .
The diff of _this_ PR without those test changes is [here](https://github.com/andrej/mlir-aie/compare/bd-chain-tests..configure-task-allow-other-bufs)